src/expression.c

```diff
@@ -3280,9 +3280,9 @@ Expression *IdentifierExp::semantic(Scope *sc)
     {
         s = sc->search_correct(ident);
         if (s)
-            error("undefined identifier %s, did you mean %s %s?", ident->toChars(), s->kind(), s->toChars());
+            error("undefined identifier '%s', did you mean '%s %s'?", ident->toChars(), s->kind(), s->toChars());
```

Comment:
I think only the identifier should be quoted here, not the kind and the identifier.